.NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs

```diff
@@ -72,6 +73,12 @@ protected override void InitializeConfiguration()
                 (options) => new DateTimeModel(
                     new BaseMergedParser(new PortugueseMergedParserConfiguration(options)),
                     new BaseMergedExtractor(new PortugueseMergedExtractorConfiguration(options))));
+
+            RegisterModel<DateTimeModel>(
+                Culture.German,

```

Comment:
Please don't push the registration of the German DateTime models until all tests are passing, ok?
We only want to expose them when that's true.

Reply:
Ok, I added it because most of it is working, except for very few issues which mostly seem to be related to bugs that other languages have too. More details in the report.

Reply:
I see many tests marked as not supported and the DateTimeModelSplitDateAndTime and DateTimeModelCalendarMode specs are empty.
I didn't have time to look into detail.
Can you also look at the Choice recognizer and the new NumberRange type?